### PR TITLE
feat(issue-30):implement event-card collection

### PR DESCRIPTION
**Pull request by yoki1234**
Closes: #30 

Developed as: 
image

### CODE REVIEW

File: src/type.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface EventCardProps {
+  content: string;
+  imageURL?: string;
+  link: string;
+  title: string;
+  variation?:string;
+}
```
Comment (Macdeal) [inline src/type.ts:7]: change `EventCardProps`  to `EventCardData`

File: src/container/EventCard.tsx [hunk 2]
```diff
@@ -1,26 +1,24 @@
 import EventTitle from '../components/eventTitle/EventTitle';
-import { EventVariation } from '../data';
 
-interface EventCardProps {
-  content: string;
-  imageURL: string;
-  link: string;
-  title: string;
-}
```
Comment (Macdeal) [inline src/container/EventCard.tsx:9]: revert this, props type should be in same file

File: src/type.ts (hunk 1 above)
Comment (Macdeal) [inline src/type.ts:7]: remove this

File: src/container/EventCard.tsx [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
+  imageURL,
+  link,
+  title,
+  variation,
+}: EventCardProps) => (
+    <div className="w-full h-full p-4 lg:p-8 flex flex-1 flex-col justify-between rounded-lg border border-gray-800 bg-gray-900">
+      <div className="bg-gray-800 h-48 flex items-center justify-center rounded mb-6 p-4">
+        <img className="w-32 max-h-24" src={imageURL} alt={title} />
+      </div>
+      <EventTitle text={title} variation={variation || 'card-title'} />
```
[the remark this answers is not in the document]
Comment (yoki1234) [reply src/container/EventCard.tsx:24]: explained it in Todo if any concern do let me know

**Review by APJ-9 — CHANGES_REQUESTED** [review]
Please check all the component variations  widths
image

**Review by APJ-9 — APPROVED** [review]

**Review by Musthaq-Ahmad — APPROVED** [review]

**Review by Macdeal — APPROVED** [review]

**Pull request merged**